src/sugar3/graphics/toggletoolbutton.py

```diff
@@ -28,6 +27,12 @@
 from sugar3.graphics.icon import Icon
 from sugar3.graphics.palette import Palette, ToolInvoker
 
+'''
+Toogle tool button switches you between two tasks.
+For example, a button can contain two functions like
+On or Off.
```

Comment:
I still don't think "tasks" and "on or off" is a good example.  If you feel you must use an example to explain this Sugar ToggleToolButton because it is different to Gtk.ToggleToolButton, then use the Journal favourites selection button as a guide.